en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -80,308 +74,302 @@ See the instructions on [configuring the API Gateway]({{base_path}}/api-analytic
+    http_endpoint = "http://[api-gateway-LB-host]"
+    https_endpoint = "https://[api-gateway-LB-host]"
+
+    ```
+
+    ```toml tab="Single Gateway"
+
+    [[apim.gateway.environment]]
+    name = "Default"
+    type = "hybrid"
+    display_in_api_console = true
+    description = "This is a hybrid gateway that handles both production and sandbox token traffic."
+    show_as_token_endpoint_url = true
+    service_url = "https://[api-gateway-host]:9443/services/"
+    ws_endpoint = "ws://[api-gateway-host]:9099"
+    wss_endpoint = "wss://[api-gateway-host]:8099"
+    http_endpoint = "http://[api-gateway-host]:${http.nio.port}"
+    https_endpoint = "https://[api-gateway-host]:${https.nio.port}"
+    
+    ```
+
+3. Optionally, add the following configuration to enable distributed cache invalidation within the nodes.
+
+    ```toml
+    
+    [apim.cache_invalidation]
+    enabled = true
+    domain = "control-plane-domain"
+    ```
+
+4. Follow the steps given below to configure High Availability (HA) for the Control Plane:
+
+    1. Create a copy of the API-M Control Plane node that you just configured. This is the second node of the API-M Control Plane cluster.
+
+    2. Configure a load balancer fronting the two Control Plane nodes in your deployment. For instructions, see [Configuring the Proxy Server and the Load Balancer]({{base_path}}/install-and-setup/setup/setting-up-proxy-server-and-the-load-balancer/configuring-the-proxy-server-and-the-load-balancer/).
+
+#### Sample configuration for the Control Plane
+
+```toml
+
+[server]
+hostname = "cp.wso2.com"
+node_ip = "127.0.0.1"
+server_role = "control-plane"
+offset=0
+
+[user_store]
+type = "database_unique_id"
+
+[super_admin]
+username = "admin"
+password = "admin"
+create_admin_account = true
+
+[database.apim_db]
+type = "mysql"
+hostname = "db.wso2.com"
+name = "apim_db"
+port = "3306"
+username = "root"
+password = "root"
+
+[database.shared_db]
+type = "mysql"
+hostname = "db.wso2.com"
+name = "shared_db"
+port = "3306"
+username = "root"
+password = "root"
+
+[keystore.tls]
+file_name =  "wso2carbon.jks"
+type =  "JKS"
+password =  "wso2carbon"
+alias =  "wso2carbon"
+key_password =  "wso2carbon"
+
+# Gateway configuration
+[[apim.gateway.environment]]
+name = "Default"
+type = "hybrid"
+display_in_api_console = true
+description = "This is a hybrid gateway that handles both production and sandbox token traffic."
+show_as_token_endpoint_url = true
+service_url = "https://gw.wso2.com:9443/services/"
+username= "${admin.username}"
+password= "${admin.password}"
+ws_endpoint = "ws://gw.wso2.com:9099"
+wss_endpoint = "wss://gw.wso2.com:8099"
+http_endpoint = "http://gw.wso2.com:8280"
+https_endpoint = "https://gw.wso2.com:8243"
+
+[apim.cors]
+allow_origins = "*"
+allow_methods = ["GET","PUT","POST","DELETE","PATCH","OPTIONS"]
+allow_headers = ["authorization","Access-Control-Allow-Origin","Content-Type","SOAPAction"]
+allow_credentials = false
+```
```

Comment:
Let's remove the Traffic Manager startup command section from this document.

Reply:
Should I remove all these three configurations?

Reply:
Removed TM startup commands.